.gitignore

```diff
@@ -0,0 +1,5 @@
+.nyc_output
```

Comment:
do we want spaces between these (also looking at the .npmignore)

Reply:
I see that we do that in places, but I don't think it adds value on such a small file.  I think it's more readable as a small block without comments (since they're mostly self-explanatory)